src/components/createRecipe/recipeForm/Step5.jsx

```diff
@@ -67,9 +67,9 @@ function Step5(props) {
       {recipe.description}
       </p>
       <h2>Ingredients</h2>
-        {recipe_ingredients.map(ingr => <p>{ingr.name}</p>)}
+        {recipe_ingredients.map((ingr, i) => <p key={i}>{ingr.name}</p>)}
```

Comment:
we may have merged issues here because I modify this part too

Reply:
In the same way? :)